extensions/src/main/java/org/pentaho/platform/web/http/filters/PentahoWebContextFilter.java

```diff
@@ -60,6 +67,13 @@
   private static final String REQUIREJS_CONFIG_LOCATION = "content/common-ui/resources/web/require-cfg.js";
   private static final String REQUIREJS_INIT_LOCATION = "osgi/requirejs-manager/js/require-init.js";
 
+  static final String DEFAULT_OSGI_BRIDGE = "osgi/";
```

Comment:
We were discussing this after you left and we had doubts about this, so we decided to merge and if tomorrow we find that the urls should go with '/' at the beginning, I will do a second round of commits to change it